controllers/comments.controller.js

```diff
@@ -0,0 +1,27 @@
+const { checkArticleExists } = require("../db/seeds/utils");
+const {
+  fetchCommentsById,
+  postCommentsById,
+} = require("../models/comments.model");
+
+exports.getCommentsById = async (req, res, next) => {
+  try {
+    const { article_id } = req.params;
+    const comments = await fetchCommentsById(article_id);
+    return res.status(200).send({ comments });
+  } catch (err) {
+    next(err);
+  }
+};
+
+exports.postComments = async (req, res, next) => {
+  try {
+    const newComment = req.body;
+    const { article_id } = req.params;
+    const checkArticle = await checkArticleExists(article_id);
```

Comment:
We likely won't need to check the article exists first, as this is a foreign key psql will throw an error if you try to insert onto an article that doesn't exist.